docs/install-agent.md

```diff
@@ -83,84 +83,66 @@ If you missed it in the "Provision a Server" page, we recommend you use Ubuntu 2
+    Use the `install-service` command to configure and install a systemd service to start the actuated agent.
 
-    In addition, any bootstrap tokens sent to the agent are further encrypted with the agent's public key.
+    The actuated control plane will only communicate with agents exposed over HTTPS to ensure proper encryption is in place. An API token is used in addition with the TLS connection for all requests.
+
+    Any bootstrap tokens sent to the agent are further encrypted with the agent's public key.
 
     For hosts with public IPs, you will need to use the built-in TLS provisioning with Let's Encrypt. For hosts behind a firewall, NAT or in a private datacenter, you can use inlets to create a secure tunnel to the agent.
 
     We're considering other models for after the pilot, for instance GitHub's own API has the runner make an outbound connection and uses long-polling.
 
-    See also: [expose the agent with HTTPS](/expose-agent/)
-
-5. Start the agent
-
-    We suggest starting the agent through a script or tmux session initially, then once we've confirmed it's enrolled correctly, switch over to systemd.
+    These steps are for hosts with public IP addresses, if you want to [use inlets](/expose-agent/), jump to the end of this step.
 
     The easiest way to configure everything is to run as root, however, you can also use a non-root user with passwordless `sudo`, if you prefer.
 
-    These steps are for hosts with public IP addresses, if you want to [use inlets](/expose-agent/), jump to the end of this step.
-
-    For an *x86_64* server, add the following to: `/root/start.sh`:
+    For an *x86_64* server, run:
 
     ```bash
-    #!/bin/bash
-
-    echo Running Agent from: ./agent
     DOMAIN=agent1.example.com
 
-    sudo -E agent up \
-        --image-ref=ghcr.io/openfaasltd/actuated-ubuntu20.04:x86-64-latest \
-        --kernel-ref=ghcr.io/openfaasltd/actuated-kernel-5.10.77:x86-64-latest \
-        --letsencrypt-domain $DOMAIN \
-        --letsencrypt-email webmaster@$DOMAIN
+    sudo -E agent install-service \
+      --letsencrypt-domain $DOMAIN \
+      --letsencrypt-email webmaster@$DOMAIN \
+      --user root
     ```
 
-    For an Arm64 server, add the following to `/root/start.sh` instead: 
+    For an *Arm64* server, run: 
 
     ```bash
-    #!/bin/bash
-
-    echo Running Agent from: ./agent
     DOMAIN=agent1.example.com
 
-    sudo -E agent-arm64 up \
-        --image-ref=ghcr.io/openfaasltd/actuated-ubuntu20.04:aarch64-latest  \
-        --kernel-ref=ghcr.io/openfaasltd/actuated-kernel-5.10.77:aarch64-latest \
-        --letsencrypt-domain $DOMAIN \
-        --letsencrypt-email webmaster@$DOMAIN
+    sudo -E agent-arm64 install-service \
+      --letsencrypt-domain $DOMAIN \
+      --letsencrypt-email webmaster@$DOMAIN \
+      --user root
```

Comment:
Isn't `root` the default? Can we omit it then?

Reply:
Root is the default and it can be omitted. I will just add a note that the `--user` flag can be used to run as a different user.

Reply:
> The `--user` flag can be used to run under a custom user account, however sudo access is still required for actuated.